.github/workflows/django.yaml

```diff
@@ -32,6 +32,10 @@ on:
         required: false
         type: boolean
         default: false
+      codeCoverage:
+        required: false
+        type: boolean
+        default: true
       # Dependency manager config
```

Comment:
Why is this added? Also, are we sure about updating `django.yaml` and not the other backend actions file?

Reply:
Makes it easier to update projects to latest shared-actions that don't already use codeCoverage (even for just testing).

All products I looked at use `django.yaml`